kikar_hamedina/templates/layouts/old-generic-template.html

```diff
@@ -7,7 +7,7 @@
 
     <!-- Fonts -->
     <link href="//netdna.bootstrapcdn.com/font-awesome/4.0.3/css/font-awesome.css" rel="stylesheet">
-    <link href='http://fonts.googleapis.com/css?family=Alef:400,700&subset=all' rel='stylesheet' type='text/css'>
+    <link href='//fonts.googleapis.com/css?family=Alef:400,700&subset=all' rel='stylesheet' type='text/css'>
```

Comment:
Not sure why this is relevant. There doesn't seem to be any usage of this template anywhere.

Reply:
removed it.